frontend/src/App.js

```diff
@@ -7,26 +7,18 @@ import UCSBDatesIndexPage from "main/pages/UCSBDates/UCSBDatesIndexPage";
 import UCSBDatesCreatePage from "main/pages/UCSBDates/UCSBDatesCreatePage";
 import UCSBDatesEditPage from "main/pages/UCSBDates/UCSBDatesEditPage";
 
```

Comment:
App.js must include changes from main. Consider rebasing off of main to include these changes.